engines/director/castmember.cpp

```diff
@@ -1036,11 +1036,6 @@ void TextCastMember::importRTE(byte *text) {
 }
 
 void TextCastMember::setText(const Common::U32String &text) {
-	// Do nothing if text did not change
```

Comment:
Removing this as is will lead to big overhead: text could be set at every frame step.

What you could do instead, is emptying _ptext with `_ptext.clear()` before calling `setText()` in `the textFont` and other setters, thus, this particular condition would be skipped.